### PR TITLE
feat: Added Alert and Button Component.

### PR DESCRIPTION
**Pull request by xii-vi**
# Alert
#### Alert with icons
Screenshot (22)

# Button
#### Buttons with icon

Screenshot (21)

### CODE REVIEW

File: RA-One.css [hunk 1]
```diff
@@ -1,13 +1,11 @@
-@import "/stylesheet/main.css";
-
 :root{
-    ---background-color:rgba(255, 255, 255, 0.938);
+    ---background-color: linear-gradient(to top,#0e6aa86c, #e8f1f27e);
 }
```
Comment (haarsh24) [inline RA-One.css:3]: Use the same color codes everywhere

File: stylesheet/button.css [hunk 2]
```diff
@@ -1,45 +1,131 @@
...
+  background: var(--primary-color);
+}
+.plus{
+  background-color: var(--black);
+  color: var(--text-color);
+}
+.minus{
+  color: var(--black);
+  background-color: var(--white);
+}
```
Comment (haarsh24) [inline stylesheet/button.css:35]: Give good class name like plus minus icon

**Review by haarsh24 — APPROVED** [review]

File: RA-One.css (hunk 1 above)
Comment (xii-vi) [reply RA-One.css:3]: _Suggested changes are made_

File: stylesheet/button.css (hunk 2 above)
Comment (xii-vi) [reply stylesheet/button.css:35]: _Suggested changes are made_

**Review by darshandamre — APPROVED** [review]
looks fine, you can merge

**Pull request merged**